content/en/blog/2021-03-19-a-parrot-trainer-eats-crow.md

```diff
@@ -5,7 +5,7 @@ date: "2021-03-19T08:42:21-04:00"
 draft: false
 image: img/blog/2021-03-19-a-parrot-trainer-eats-crow/parrots.jpg
 author: Rebecca Bilbro
-category: AI/ML
+tags: ['AI/ML']
```

Comment:
The page for this tag will appear as `/tags/ai/ml` in the browser. Should we change this to `AI-ML` or separate list it as `['AI', 'ML']`?

Reply:
Great point - yes, let's split into 2 tags: ['AI', 'ML']